apps/demo-typegen/script/Forc.toml

```
@@ -0,0 +1,7 @@
+[project]
+authors = ["FuelLabs"]
```

```suggestion
authors = ["Fuel Labs <contact@fuel.sh>"]
```

Super minor and I know you are updated it, but for consistency it should be like this ☝️

---

Which one should we go for everywhere?

---

The one with the email should be it 😁

---

@digorithm gotcha, do you mind if I tackle this in a separate PR since we will need to change _all_ the `Forc.toml` files in basically all the Sway projects in the repo?

---

> do you mind if I tackle this in a separate PR since we will need to change all the Forc.toml files in basically all the Sway projects in the repo?

You can also put a check in the CI pipeline similar to [this for package.jsons](https://github.com/FuelLabs/fuels-ts/blob/dd931826d2f7dade24a459d87656fcd5dca4884a/.github/workflows/test.yaml#L28). I think all of us have made this `Forc.toml` error at one point and we should let the CI catch it.